src/tokens/utils.ts

```diff
@@ -127,8 +127,7 @@ export function getSupportedNetworkIdsForTokenBalances(): NetworkId[] {
 }
 
 export function showAssetDetailsScreen() {
-  // TODO(ACT-919): get from feature gate
-  return false
+  return getFeatureGate(StatsigFeatureGates.SHOW_ASSET_PAGE_REDESIGN)
```

Comment:
I think we've generally favored dropping the wrapper dummy function whenever a feature gate is implemented. Although I don't feel too strongly on this. At the least let's keep the util function and feature gate names the same